services/app/src/reducers/Search.tsx

```diff
@@ -1,11 +1,13 @@
 import Redux from 'redux';
-import {SearchResponseAction} from '../actions/ActionTypes';
+import {LanguageType} from 'shared/schema/Constants';
+import {SearchChangeParamsAction, SearchResponseAction} from '../actions/ActionTypes';
+import {getStorageString, setStorageKeyValue} from '../LocalStorage';
 import {SearchState} from './StateTypes';
 
 export const initialSearch: SearchState = {
-  results: [],
+  results: null, // null = need to search; [] = no results
   params: {
-    language: 'English',
+    language: getStorageString('language', 'English') as LanguageType,
```

Comment:
This should still be a file-level constant value IMO.

Reply:
Should we change how do it for settings too then? https://github.com/ExpeditionRPG/expedition/blob/master/services/app/src/reducers/Settings.tsx#L9

Reply:
Probably a good idea, yeah. Seems like we should have a `defaultSettings` const with the defaults, and then an `initialSettings` which is a map of the `defaultSettings` keys, resolving via local storage where possible.

Reply:
From IRL, defaultSettings stuff would be a lot of work, so maybe later